### ISSUE TITLE
CAPErrorMessageParameterlessImpl toString() mod

### ISSUE THREAD

**Issue by AlerantAppNGIN** [RestComm/jss7#113]
Hello @vetss 

In our fork in the toString() method of the CAPErrorMessageParamerlessImpl class we converted the errorCodes to human readable messages. I'll send  a PR for this issue so you can merge it if you like this tiny mod.

Best Regards,
Gabor

**Comment by vetss** [comment]
Hello @AlerantAppNGIN,

I have commented you PR https://github.com/RestComm/jss7/pull/114 into netty-2 branch. Thanks for your work. I have only added into toString() methods also a numeric valuu so it returns now values like
"0:canceled". 
Somebody needs also digital values. I believe it is good enouph for you too.

Fixed by:
https://github.com/RestComm/jss7/commit/d0481923d89b3030105fa179ceedc43f924592ce
https://github.com/RestComm/jss7/commit/01d4a7513961c7d4bcf00034d6940a23a7a5dc14
